templates/hiera.yaml.epp

```diff
@@ -0,0 +1,73 @@
+<%- |
```

Comment:
Wrote some tests. Please take a look and let me know if you think that we should more tests.